Tests/Blazorise.Tests/Extensions/TestExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using Bunit;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Tests.Extensions
+{
+    internal static class TestExtensions
+    {
+        private static TimeSpan waitTime = TimeSpan.FromSeconds( 3 );
+
+        public static void WaitForElementAndClick<T>( this IRenderedComponent<T> comp, string selector ) where T : IComponent
```

Comment:
Can we just have a method name `Click` that accepts the selector?

Reply:
You mean, rename the method?

Reply:
Yes.